### DISCUSSION TITLE
Images with no Figure caption

### DISCUSSION THREAD

**Discussion by ptram** [quarto-dev/quarto-cli#7036] [Q&A] [answered]
### Description

Hi,

I see several discussions about captions, but can't find anything related to my question. So, I'll ask.

I'm trying to have images with no "Figure" caption. When rendering to a Website, just leaving the caption between square parentheses empty, in the markdown source, works fine: the image is rendere with no caption.

When rendering to PDF through Typst, a 'Figure [#]:' always appears below the image. I would like it not to appear, since it is not needed in the style I'm after (a user manual). Said in another way: I don't want a caption under the images.

How to do to hide it? I've tried a few solutions found on the web, but none did work. Is there a direct way, maybe something already described in the manuals, but that I've not yet been able to find?

**Comment by cderv** [comment]
Can you share with us an example document so that we are trying to help on the same base as you ? 

Is the question only for the Typst format ?

**Reply by ptram** [reply]
I don't know if this only happens with Typst. Judging from the examples in the Quarto manual ("Figures"), it looks like this might also happen with LaTeX. I admit I've only tried Quarto with Website, Revealjs and PDF via Typst.

A very simple source markdown is this one:

```
Please note that two other ports are on the back of the instrument.
![](images/usb-f-area-detail.png){width=30% fig-alt="Detail of the 'Front USB port', on the extreme right of the control panel"}
```

As you see, the caption is empty. This translates as I want in the Website:

<img width="906" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/19186065/a43af838-c325-4d4e-bec5-484c0bbe1f27">

This is the HTML code generated by Quarto:

```
Please note that two other ports are on the back of the instrument.</p>
<div id="fig-539a35d47e664c97a50115a146a7f1bd-10" class="quarto-figure quarto-figure-center anchored" width="30%" alt="Detail of the 'Front USB port', on the extreme right of the control panel">
<figure class="quarto-float quarto-float-fig figure">
<div>
<img src="images/usb-f-area-detail.png" id="fig-539a35d47e664c97a50115a146a7f1bd-10" class="img-fluid figure-img" style="width:30.0%" alt="Detail of the 'Front USB port', on the extreme right of the control panel">
</div>
</figure>
</div>
```

The PDF generated through Typst (with Quarto 1.4.386) contains the "Figure:" statements that I don't want, even if the caption is still empty:

<img width="452" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/19186065/c986046b-49bc-437c-abd3-6b0edc7a0380">

This latest style is good for academic papers, but not for user manuals, where images don't need cross-referencing, but are only given where they are required. So, I would like that nothing of the caption to appear.

**Comment by cscheid** [comment, marked as answer]
This is a bug in our typst output! I'll fix right away.

**Reply by cderv** [reply]
* https://github.com/quarto-dev/quarto-cli/issues/7041
